packages/components/src/DataList/DataList.test.tsx

```diff
@@ -74,6 +129,20 @@ describe("DataList", () => {
     });
   });
 
+  it("should render a title when it's provided", () => {
```

Comment:
I wonder if you should test the inverse of this

Reply:
Any ideas how this could be done? Because if a title is not provided, there's nothing to search for.

Reply:
hmmm. `within` + test id? you already have the title container div haha